src/rt-app.c

```diff
@@ -300,38 +304,49 @@ static int run_event(event_data_t *event, int dry_run,
 		pthread_mutex_unlock(&(rdata->res.barrier.m_obj));
 		break;
 	case rtapp_sig_and_wait:
-		log_debug("signal and wait %s", rdata->name);
+		log_debug("[%d] sync %s", task_idx, rdata->name);
+		log_debug("[%d] sync - signal %s", task_idx, rdata->name);
```

Comment:
why do you use 2 lines ?
looks like these 2 lines display the same values ?

Reply:
Fixed this